gensrc/thrift/Descriptors.thrift

```diff
@@ -240,6 +240,10 @@ struct TOlapTable {
     1: required string tableName
 }
 
+struct TMaterializedView {
```

Comment:
why do need pass the mv name info to BE?  when do we need this thrift info in BE?

Reply:
I will remove this structure.